tests/filesystem.go

```diff
@@ -98,8 +98,8 @@ func getRootLocation(partitions []*types.Partition) string {
 }
 
 // returns true if no error, false if error
-func runIgnition(t *testing.T, ctx context.Context, stage, root, cwd string, appendEnv []string) error {
-	args := []string{"-clear-cache", "-oem", "file", "-stage", stage, "-root", root, "-log-to-stdout"}
+func runIgnition(t *testing.T, ctx context.Context, stage, root, cwd string, appendEnv []string, oemMountPointDir string) error {
```

Comment:
Maybe we should move the `oemMountPointDir` earlier in the parameters with the rest of the strings?